### PR TITLE
fix(dedicated): fix ip modal

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

add oui-kit inputs for form
fix tooltip

### Possible Drawbacks

None

### Applicable Issues

resolves MBE-111

### CODE REVIEW

File: client/app/ip/ip/order/PCC.html [hunk 1]
```diff
@@ -1,124 +1,104 @@
...
+        <label for="control-label" class="oui-label" data-translate="ip_order_networkname"></label>
+        <div class="oui-field__content">
+            <div class="oui-field-control oui-field-control_5 d-flex">
+                <input type="text" class="oui-input"
+                       data-ng-pattern="/^[a-zA-Z]+\w+$/"
+                       data-ng-model="model.params.networkName"
+                       name="networkName"
+                       id="networkName"
+                       data-maxlength="20">
+                <span class="fa fa-info-circle m-2" data-oui-tooltip="{{:: 'ip_order_networkname_help' | translate }}"></span>
```
Comment (marie-j) [inline client/app/ip/ip/order/PCC.html]:
```suggestion
                <span class="oui-icon oui-icon-info_circle oui-icon_bicolor m-2" data-oui-tooltip="{{:: 'ip_order_networkname_help' | translate }}"></span>
```
? 
Also would it make sense to replace this `tooltip` by a `oui-field` with a  `label-popover` ?

File: client/app/ip/ip/order/PCC.html [hunk 2]
```diff
@@ -1,124 +1,104 @@
...
+        <label class="oui-select">
+            <select
+                id="sizeSelect"
                 name="sizeSelect"
-                class="form-control"
                 data-ng-model="model.params.size"
-                data-ng-options="r as ('/'+r) for r in orderableIp.size">
-            <option value=""
+                class="oui-select__input"
+                data-ng-options="r as ('/'+r) for r in orderableIp.size"
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: Could you please rename `r` ? If it's a size it should be named `size`

File: client/app/ip/ip/order/PCC.html [hunk 3]
```diff
@@ -1,124 +1,104 @@
...
-
-    <div class="form-group">
-        <label class="control-label"
-               for="sizeSelect"
-               data-translate="ip_order_ipblocksize">
-        </label>
-        <select id="sizeSelect"
+    <oui-field data-label="{{:: 'ip_order_ipblocksize' | translate }}" data-size="xl">
+        <label class="oui-select">
+            <select
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: `oui-select` wouldn't work ?

File: client/app/ip/ip/order/PCC.html [hunk 4]
```diff
@@ -1,124 +1,104 @@
...
+                id="countrySelect"
                 name="countrySelect"
-                class="form-control"
                 data-ng-model="model.params.country"
-                data-ng-options="c as ('country_' + c.toUpperCase() | translate) for c in orderableIp.countries"
-                data-ng-required="true">
-            <option value=""
+                class="oui-select__input"
+                data-ng-options="c as ('country_' + c.toUpperCase() | translate) for c in orderableIp.countries"
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: Can you rename `c` to `countryName` ?

File: client/app/ip/ip/order/PCC.html [hunk 5]
```diff
@@ -1,124 +1,104 @@
...
+                </option>
+            </select>
+            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
         </label>
-        <select id="countrySelect"
+    </oui-field>
+
+    <oui-field data-label="{{:: 'ip_order_country' | translate }}" data-size="xl">
+        <label class="oui-select">
+            <select
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: `oui-select` ?

File: client/app/ip/ip/order/PCC.html (hunk 1 above)
Comment (FredericEspiau) [reply client/app/ip/ip/order/PCC.html]: A span is not `focusable`, you should not set a `tooltip` on it. You can use a `oui-popover` instead

File: client/app/ip/ip/order/PCC.html [hunk 6]
```diff
@@ -1,124 +1,104 @@
...
+        <label for="control-label" class="oui-label" data-translate="ip_order_estimatedclientsnumber"></label>
+        <div class="oui-field__content">
+            <div class="oui-field-control oui-field-control_5 d-flex">
+                <input type="number" class="oui-input"
+                       data-ng-model="model.params.estimatedClientsNumber"
+                       name="estimatedClientsNumber"
+                       id="estimatedClientsNumber"
+                       min="1"
+                       data-required>
+                <span class="fa fa-info-circle m-2" data-oui-tooltip="{{:: 'ip_order_estimatedclientsnumber_help' | translate }}"></span>
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: Same here

File: client/app/ip/ip/order/PCC.html [hunk 7]
```diff
@@ -1,124 +1,104 @@
...
+                    data-rows="3"
+                    id="description"
+                    name="description"
+                    data-maxlength="250"
+                    data-minlength="3"
+                    data-ng-pattern="/^[a-zA-Z0-9\.,\-'!\?:;\(\)\+\/=\s]*$/"
+                    data-model="model.params.description"
+                    data-required>
+                </oui-textarea>
+                <span class="fa fa-info-circle m-2" data-oui-tooltip="{{:: 'ip_order_description_help' | translate }}"></span>
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: Same here

File: client/app/ip/ip/order/PCC.html [hunk 8]
```diff
@@ -1,124 +1,104 @@
...
+    <div class="oui-field">
+        <label for="control-label" class="oui-label" data-translate="ip_order_usage"></label>
+        <div class="oui-field__content">
+            <div class="oui-field-control oui-field-control_5 d-flex">
+                <input type="text" class="oui-input"
+                       data-ng-model="model.params.usage"
+                       name="usage"
+                       id="usage"
+                       required>
+                <span class="fa fa-info-circle m-2" data-oui-tooltip="{{:: 'ip_order_usage_help' | translate }}"></span>
```
Comment (FredericEspiau) [inline client/app/ip/ip/order/PCC.html]: Same here

File: client/app/ip/ip/order/PCC.html [hunk 9]
```diff
@@ -1,124 +1,80 @@
...
+            </select>
+            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
         </label>
-        <input type="text"
-               class="form-control"
-               id="networkName"
-               name="networkName"
+    </oui-field>
+
+    <oui-field label="{{:: 'ip_order_networkname' | translate }}" label-popover="{{:: 'ip_order_networkname_help' | translate }}">
```
Comment (antleblanc) [inline client/app/ip/ip/order/PCC.html]:
```suggestion
    <oui-field data-label="{{:: 'ip_order_networkname' | translate }}" data-label-popover="{{:: 'ip_order_networkname_help' | translate }}">
```

File: client/app/ip/ip/order/PCC.html [hunk 10]
```diff
@@ -1,124 +1,80 @@
...
-         }">
-        <label class="control-label"
-               for="estimatedClientsNumber"
-               data-translate="ip_order_estimatedclientsnumber">
-        </label>
-        <input type="number"
-               id="estimatedClientsNumber"
-               name="estimatedClientsNumber"
-               class="form-control"
+    <oui-field label="{{:: 'ip_order_estimatedclientsnumber' | translate }}" label-popover="{{:: 'ip_order_estimatedclientsnumber_help' | translate }}">
```
Comment (antleblanc) [inline client/app/ip/ip/order/PCC.html]:
```suggestion
    <oui-field data-label="{{:: 'ip_order_estimatedclientsnumber' | translate }}" data-label-popover="{{:: 'ip_order_estimatedclientsnumber_help' | translate }}">
```

File: client/app/ip/ip/order/PCC.html [hunk 11]
```diff
@@ -1,124 +1,80 @@
...
-                  data-ng-required="true"
-                  style="resize: none;"
-                  data-ng-model="model.params.description">
-        </textarea>
-        <i class="fa fa-info-circle"
-           data-tooltip-placement="left"
-           data-uib-tooltip="{{ 'ip_order_description_help' | translate }}">
-        </i>
-    </div>
+    <oui-field label="{{:: 'ip_order_description' | translate }}" label-popover="{{:: 'ip_order_description_help' | translate }}">
```
Comment (antleblanc) [inline client/app/ip/ip/order/PCC.html]:
```suggestion
    <oui-field data-label="{{:: 'ip_order_description' | translate }}" data-label-popover="{{:: 'ip_order_description_help' | translate }}">
```

File: client/app/ip/ip/order/PCC.html [hunk 12]
```diff
@@ -1,124 +1,80 @@
...
-         }">
-        <label class="control-label"
-               for="usage"
-               data-translate="ip_order_usage">
-        </label>
-        <input type="text"
-               id="usage"
-               name="usage"
-               class="form-control"
+    <oui-field label="{{:: 'ip_order_usage' | translate }}" label-popover="{{:: 'ip_order_usage_help' | translate }}">
```
Comment (antleblanc) [inline client/app/ip/ip/order/PCC.html]:
```suggestion
    <oui-field data-label="{{:: 'ip_order_usage' | translate }}" data-label-popover="{{:: 'ip_order_usage_help' | translate }}">
```

**Pull request merged**